.github/ISSUE_TEMPLATE/support.yml

```diff
@@ -0,0 +1,37 @@
+name: "Usage question 🙋🏽‍♀️"
+description: "Create a report to help us fix something that is currently broken."
+title: "[Q/A] - <title>"
+labels: ["type: question 🤔"]
```

Comment:
Can we just redirect this to GH discussions? The issue tracker is not well suited for usage questions.

Reply:
Yep, will update and enable discussions.

I created this because questions in this stage are likely to be bugs/feature-requests/documentation-blindspots, which can be easier to track with issues. I don't feel super strongly about this though, so happy to change. :)